web/src/scheduled_messages.js

```diff
@@ -0,0 +1,158 @@
+import $ from "jquery";
+
+import render_scheduled_message from "../templates/scheduled_message.hbs";
+import render_scheduled_messages_overlay from "../templates/scheduled_messages_overlay.hbs";
+
+import * as blueslip from "./blueslip";
+import * as browser_history from "./browser_history";
+import * as channel from "./channel";
+import * as compose from "./compose";
+import * as compose_actions from "./compose_actions";
+import * as compose_fade from "./compose_fade";
+import * as compose_ui from "./compose_ui";
+import * as loading from "./loading";
+import * as narrow from "./narrow";
+import * as overlays from "./overlays";
+import * as people from "./people";
+import * as stream_color from "./stream_color";
+import * as stream_data from "./stream_data";
+import * as timerender from "./timerender";
+
+// This is only updated when user opens the scheduled messages overlay.
+let scheduled_messages_data = [];
+
+function hide_loading_indicator() {
+    loading.destroy_indicator($("#scheduled_messages_overlay .loading-indicator"), {
+        abs_positioned: true,
+    });
```

Comment:
I don't think `loading.destroy_indicator` takes a second parameter?

Reply:
Yeah, we might have changed the implementation from when this code was written. Nice catch.